server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -578,6 +610,19 @@ public EventStream getEventStream() {
          return eventStream;
       }
 
+      @CacheEntryCreated
+      public CompletionStage<Void> onEntryCreated(CacheEntryCreatedEvent<?, EventLog> entry) {
+         if (!(entry instanceof EventImpl) || entry.isPre()) return CompletableFutures.completedNull();
```

Comment:
Why does the event have to be an instance of `EventImpl`? We never use any implementation specific methods, so any `EventLog` implementation should be fine. My understanding is that we would want to include `ClusterEvent` for events that did not originate on the node that was being listened to.

Reply:
Yeah, verifying here it is not really necessary. I started the initial implementation using it and just kept :/